.gitignore

```diff
@@ -111,15 +111,13 @@ ENV/
 # alembic
 # alembic/
 # alembic.ini
-
 # pytests
 .pytest_cache/
 html_coverage_report/
 *.DS_store
 
 # credentials
 credentials.json
-
 #logs
```

Comment:
If there is nothing of any significance to change here, let's have the new lines removed. It becomes a redundant change